region/brinstar/green.json

```diff
@@ -4177,6 +4291,98 @@
                     "Build up some run speed and then extend the Grapple Beam through the Blue Gate, while jumping, to open it from the wrong side.",
                     "This may take many attempts."
                   ]
+                },
+                {
+                  "name": "G-Mode Indirect Despawn Gate",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [3],
+                      "mode": "indirect",
+                      "artificialMorph": false
+                    }}
+                  ]
+                }
+              ]
+            },
+            {
+              "id": 2,
+              "strats": [
+                {
+                  "name": "G-Mode Morph Indirect Despawn Gate - To the Top Right Door",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [3],
+                      "mode": "indirect",
+                      "artificialMorph": true
+                    }},
+                    {"or": [
+                      "canTrickyJump",
+                      {"and": [
+                        "Bombs",
+                        {"ammo": { "type": "PowerBomb", "count": 1 }}
+                      ]},
+                      {"ammo": { "type": "PowerBomb", "count": 3 }}
+                    ]},
+                    {"or": [
+                      "h_canArtificialMorphIBJ",
+                      "h_canArtificialMorphSpringBallBombJump"
+                    ]}
+                  ],
+                  "note": [
+                    "Use a Power Bombs to kill the first three hoppers,",
+                    "or very carefully dodge them by quickly rolling between the first two tall flowers and waiting for the hopper to jump over Samus.",
+                    "The middle hopper can be killed safely with many Bombs by staying on the top left of the raised ground,",
+                    "without overhanging or the hopper can hit Samus. Alternatively, lure and spring ball jump past it.",
+                    "Killing the bugs in their pipes is possible without getting hit, or use spring ball to jump past them.",
+                    "Once the bugs are killed once they will stop respawning.",
+                    "The last hopper can be carefully killed with Bombs, or killed with a Power Bomb.",
+                    "IBJ or bomb boost spring ball jump up to the top right door."
+                  ],
+                  "devNote": "The canTrickyJump is used for rolling into a precise location, dodging hoppers while rolling, and spring ball jumping past enemies."
```

Comment:
I found dodging the hoppers to be easier than killing them with bombs.  Takes some practice either way, to learn enemy patterns and how to deal with pipe bugs.  The top hopper I dealt with by letting it jump off to the right.